editor/plugins/script_editor_plugin.cpp

```diff
@@ -2182,7 +2182,7 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 
 	script_list = memnew(ItemList);
 	list_split->add_child(script_list);
-	script_list->set_custom_minimum_size(Size2(150 * EDSCALE, 100)); //need to give a bit of limit to avoid it from disappearing
+	script_list->set_custom_minimum_size(Size2(0, 100)); //need to give a bit of limit to avoid it from disappearing
```

Comment:
I guess you're right but the comment is not really clear about that. [When it was added](https://github.com/godotengine/godot/commit/2a3e00c8c7e6f997b12864755a3df3b9bd3cca05#diff-5b09686358173e1d947a01511212a3ebR2182) the previous minimum size was (0, 0).